setup.bat

```diff
@@ -0,0 +1 @@
+mvn package
```

Comment:
Bash is on Windows, although it is called Powershell, and does not have nearly any features you would expect. I just prefer to have the ability to type `setup` rather than `mvn package`.